### PR TITLE
fix(demo): simple tab navigator example

### PR DESCRIPTION
**Pull request by flochtililoch**
- Move recently added bottom-tabs example under community
- Rename simple-tab into bottom-tab
- Bring styles back from previous code

### CODE REVIEW

File: demo/backend/advanced/community/elements/bottom-tabbar/tab-bar.xml.njk
Comment (hgray-instawork) [inline demo/backend/advanced/community/elements/bottom-tabbar/tab-bar.xml.njk:1]: should replace the "navigate" behavior target with href hash syntax

File: demo/backend/navigation/navigator/bottom-tab/tab-1.xml.njk [hunk 1]
```diff
@@ -25,10 +29,10 @@ hv_button_behavior: "back"
       target="tab-2"
```
Comment (hgray-instawork) [inline demo/backend/navigation/navigator/bottom-tab/tab-1.xml.njk:29]: switch to href

File: demo/backend/navigation/navigator/bottom-tab/tab-2.xml.njk [hunk 2]
```diff
@@ -16,17 +20,17 @@ hv_button_behavior: "back"
       target="tab-1"
```
Comment (hgray-instawork) [inline demo/backend/navigation/navigator/bottom-tab/tab-2.xml.njk:20]: switch to href

**Review by hgray-instawork — APPROVED** [review]
Thanks!

File: demo/backend/navigation/navigator/bottom-tab/tab-2.xml.njk (hunk 2 above)
Comment (flochtililoch) [reply demo/backend/navigation/navigator/bottom-tab/tab-2.xml.njk]: [bdba9be](https://github.com/Instawork/hyperview/pull/969/commits/bdba9be7457df7c236cfc745aa08ef9f318dbeb8)

File: demo/backend/navigation/navigator/bottom-tab/tab-1.xml.njk (hunk 1 above)
Comment (flochtililoch) [reply demo/backend/navigation/navigator/bottom-tab/tab-1.xml.njk]: [bdba9be](https://github.com/Instawork/hyperview/pull/969/commits/bdba9be7457df7c236cfc745aa08ef9f318dbeb8)

File: demo/backend/advanced/community/elements/bottom-tabbar/tab-bar.xml.njk
Comment (flochtililoch) [reply demo/backend/advanced/community/elements/bottom-tabbar/tab-bar.xml.njk:1]: [bdba9be](https://github.com/Instawork/hyperview/pull/969/commits/bdba9be7457df7c236cfc745aa08ef9f318dbeb8)

**Pull request merged**